src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+$space: 1px;
+$size__box-day: 100px;
+$width-calendar: $size__box-day * 7 + $space * 6;
+$days: "mon" 0 , "tue" 1 , "wed" 2 , "thu" 3 , "fri" 4 ,"sat" 5 , "sun" 6;
+$first-day-margin: $size__box-day + $space;
+
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space;
+  max-width: $width-calendar;
+  padding: 10px;
+
+  &__item {
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: $size__box-day;
+    height: $size__box-day;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: 0.5s ease-in;
```

Comment:
It would be great to make transition duration a variable, as it should be the same for the whole page. Also, specify the styles for which you apply the transition 
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/e554e30d-205e-4976-b69a-57aaa55281a7)